Puppetfile

```diff
@@ -38,6 +38,8 @@ mod 'puppetlabs/mailalias_core', '1.0.5'
 mod 'puppetlabs/docker', '3.8.0'
 mod 'puppetlabs/reboot', '2.2.0'
 mod 'puppet/python', '3.0.1'
+mod 'derdanne/nfs', '2.1.2'
+mod 'puppet-posix_acl', '1.0.0'
```

Comment:
This is a nit-pick but I would like to consistently use `/` or `-` as the separator.

Reply:
I've reverted this commit, please take another look.